### PR TITLE
BIGTOP-4189: Mock Data Implementation for AI Chat API in Bigtop-Manager

### PR DESCRIPTION
**Pull request by lhpqaq**
https://issues.apache.org/jira/browse/BIGTOP-4189

API Doc: https://apifox.com/apidoc/shared-1b4f589f-27ef-4bb6-9c9c-e3680d3fca79/api-205377679

Commits: https://github.com/lhpqaq/bigtop-manager/commits/aichat/mock?author=lhpqaq

### CODE REVIEW

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/DateTimeUtils.java
Comment (kevinw66) [inline bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/DateTimeUtils.java:1]: We have DateUtils in common module

**Comment by kevinw66** [comment]
Please fix the ci

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/DateTimeUtils.java
Comment (lhpqaq) [reply bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/DateTimeUtils.java:1]: I have already deleted it

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/controller/AIChatController.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    @Operation(summary = "new threads", description = "Create a chat threads")
+    @PutMapping("/platforms/{platformId}/threads")
+    public ResponseEntity<ChatThreadVO> createChatThreads(@PathVariable Long platformId, @RequestParam String model) {
+        return ResponseEntity.success(chatService.createChatThreads(platformId, model));
+    }
+
+    @Operation(summary = "delete threads", description = "Delete a chat threads")
+    @DeleteMapping("platforms/{platformId}/threads/{threadId}")
+    public ResponseEntity<Integer> deleteChatThreads(@PathVariable Long platformId, @PathVariable Long threadId) {
```
Comment (lhpqaq) [inline bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/controller/AIChatController.java:94]: I’m not quite sure what the most appropriate return format here is—an empty `success()`, a `code`, or a `boolean`.

Comment (kevinw66) [reply bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/controller/AIChatController.java:94]: `ResponseEntity<Boolean>` is fine

**Review by kevinw66 — APPROVED** [review]
+1 LGTM, thanks @lhpqaq

**Pull request merged**